File: SOC/src/cc/game/soc/ui/UIRenderer.java
package cc.game.soc.ui;

import cc.game.soc.core.StringResource;
import cc.lib.ui.UIComponent;

/**
 * Created by chriscaron on 2/27/18.
 */

public abstract class UIRenderer extends cc.lib.ui.UIRenderer implements StringResource {

    UIRenderer(UIComponent component) {
        this(component, true);
    }

    UIRenderer(UIComponent component, boolean attach) {
        super(component, attach);
    }
}

File: SOC/src/cc/game/soc/ui/UIRenderer.kt
package cc.game.soc.ui

import cc.game.soc.core.StringResource
import cc.lib.ui.UIComponent
import cc.lib.ui.UIRenderer

/**
 * Created by chriscaron on 2/27/18.
 */
abstract class UIRenderer @JvmOverloads internal constructor(component: UIComponent?, attach: Boolean = true) : UIRenderer(component, attach), StringResource {
	open fun reset() {}
}